guides/doc-Troubleshooting_Guide/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::common/attributes.adoc[]
+:imagesdir: images
+include::common/header.adoc[]
+
+:deploying-on-aws:
+:context: troubleshooting-guide
+
+= Troubleshooting {ProjectName}
+
+include::common/modules/services.adoc[leveloffset=+1]
+
+include::common/modules/logging.adoc[leveloffset=+1]
+
+ifeval::["{build}" != "foreman-deb"]
+include::topics/AWS-registering-hosts-using-the-bootstrap-script.adoc[leveloffset=+1]
+endif::[]
```

Comment:
My plan for this was not to burden @chris1984 with the breakdown of the information, and to implement the tech docs structure myself afterwards and then it would be easier for him and for others to contribute within the lines we draw. I need to make something like a "new guide starter kit" and provide templates. Before I do that, we can't really blame people for not knowing how to structure guides. 
I haven't conducted a full review of the content yet, and will get to that asap.  

The log-related content looks like it would be better to add here: https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Logging_and_Reporting_Problems although I could add it to both places. 

Every time I look at the docs, I see more errors with attributes etc "Foreman 6.9 beta". I think I will need to take a full day for docs sometime soon and try and get rid of all these stupid small errors.

Reply:
Lovely! :-)